src/PKSim.Infrastructure/ORM/Queries/ModelObserverQuery.cs

```diff
@@ -148,25 +148,35 @@ private void createFractionOfDoseObserverForCompound(Simulation simulation, IObs
          //always add complex molecules if any
          var moleculeNames = processes.SpecificBindingSelection.AllEnabledPartialProcesses().Select(x => x.ProductName(CoreConstants.Molecule.Complex)).ToList();
 
-         //Add sink metabolite
-         moleculeNames.AddRange(processes.MetabolizationSelection.AllEnabledPartialProcesses().OfType<EnzymaticProcessSelection>()
+         //Add sink metabolites
+         var enzymaticProcessProductNames = processes.MetabolizationSelection.AllEnabledPartialProcesses().OfType<EnzymaticProcessSelection>()
             .Where(x => x.IsSink)
-            .Select(x => x.ProductName()));
+            .Select(x => x.ProductName()).ToList();
+
+         moleculeNames.AddRange(enzymaticProcessProductNames);
 
          if (!moleculeNames.Any())
             return;
 
+         //Local fraction of dose observer defined per instance of the metabolization or specific binding process
          var observerName = CoreConstants.Observer.ObserverNameFrom(CoreConstants.Observer.FRACTION_OF_DOSE, compound.Name);
-         var fractionObserver = createFractionObserver(observerName, observerBuildingBlock, () => getFractionOfDoseFormula(observerBuildingBlock, compound));
-
+         var fractionObserver = createAmountFractionObserver(observerName, observerBuildingBlock, () => getFractionOfDoseFormula(observerBuildingBlock, compound));
          moleculeNames.Each(fractionObserver.AddMoleculeName);
+
+         //Add liver zone specific observers based on this observer
          addLiverZoneObserversBasedOn(fractionObserver, observerBuildingBlock, compound);
+
+         //Global fraction of dose observers. One per complex or binding process defined globally under organism
+         var totalFractionObserverName = CoreConstants.Observer.ObserverNameFrom(CoreConstants.Observer.TOTAL_FRACTION_OF_DOSE, compound.Name);
```

Comment:
This is the new observer instance. One for each compound with at least a sink process OR a complex process (was not sure about complex but why not...we do it already for fraction of dose. It's probably not different)